src/assignments/scrabble-scorer.rst

```diff
@@ -0,0 +1,336 @@
+   displays a score. Each letter within the word is worth 1 point.
+#. **Bonus Vowels:** Define a method that takes a word as a parameter and
+   displays a score. Each vowel within the word is worth 3 points, and each
+   consonant is worth 1 point.
+#. **Scrabble Score:** Define a method that takes a word as a parameter and displays a score. 
+   The method uses the *transformed* ``newPointStructure``.
+#. Each method should have a way to display the *original* word and the total value of points.
+   (*Consider:* how many word parameters will you be working with?)
+
+.. admonition:: Examples
+
+   .. sourcecode:: csharp
+
+      ScrabbleScorer("taxi");
+      //Output: Your word: taxi is worth 11
+
+      SimpleScorer("taxi");
+      //Output: Your word: taxi is worth 4
+
+      BonusVowels("taxi");
+      //Output: Your word: taxi is worth 8
+
+
+
+
+Define ScoringAlgorithms method
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Create an method that will pass the user's word to the selected scoring method.
+   This method will need parameters for which score method they select and the user's word.
+#. Use the selected algorithm to determine the score for the word:
+
+   a. If the user enters ``1`` or an invalid option, use the Scrabble Scorer method.
+   b. If the user entered ``2``, use the Simple Score method.
+   c. If the user entered ``3``, use the Bonus Vowels method.
+
+#. Things to Ponder:
+
+   a. Think about what each method's parameter requirement(s).
+   b. Think about how to invoke the desired scoring method.
+
+
+Take a Breath
+~~~~~~~~~~~~~~
+
+Good! Your ``ScoringAlgorithms`` method now invokes all of the scoring
+options and passes the word to score to each.  Looks like we have a good start on this Scrabble program.
+
+To access a scoring method via ``ScoringAlgorithms``,  call it and pass the score option and a test word.
+In our example, we will be using "taxi" as our test word.
+
+.. admonition:: Examples
+
+   .. sourcecode:: csharp
+
+      // Scrabble scoring
+      ScoringAlgorithms(1, "taxi");
+         //Output: Your score for "taxi": 11
+
+      // Simple scoring
+      ScoringAlgorithms(2, "taxi");
+         //Output: Your score for "taxi": 4
+
+      // Bonus Vowel scoring
+      ScoringAlgorithms(3, "taxi");
+         //Output: Your score for "taxi": 8
+
+
+D) Tie it All Together
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Define a ``RunProgram`` method that will do the following when called:
+
+#. Invoke ``InitialPrompt`` to pick the algorithm.
+#. Prompt the user to enter a word to score. The prompt should also provide the
+   user an option for ending the program.
+#. When taking in a user's word, should account for case.
+#. Invoke the ``ScoringAlgorithms`` method chosen by the user with the correct parameters.
+#. Display the score for the word.
+#. Repeat steps 2 to 4 until the user ends the program by entering ``"Stop"``.
+   (*Consider*: Should this check be case-insensitive?)
```

Comment:
i think we need to just tell them to make this case-insensitive. otherwise its not clear what the requirement is